dev/cognitiveservices/data-plane/Language/generate-questionanswers.json

```diff
@@ -0,0 +1,643 @@
+          "type": "string",
+          "$ref": "#/definitions/QnAGenDocumentType"
+        },
+        "language": {
+          "type": "string",
+          "description": "(Optional) This is the 2 letter ISO 639-1 representation of a language. For example, use \"en\" for English; \"es\" for Spanish etc. If not set, use \"en\" for English as default."
+        }
+      },
+      "description": "Contains an input document to be analyzed by the service."
+    },
+    "QnAGenDocumentType": {
+      "type": "string",
+      "description": "QnA Gen Document type.",
+      "enum": [
+        "Raw",
+        "URL",
+        "HTML"
+      ],
+      "x-ms-enum": {
+        "modelAsString": true,
+        "name": "qnaGenDocumentType",
+        "values": [
+          {
+            "value": "Raw",
+            "description": "Raw text which can have entity markers."
+          },
+          {
+            "value": "URL",
+            "description": "Content is a URL, whose page needs to be extracted and used for QnA generation."
+          },
+          {
+            "value": "HTML",
+            "description": "HTML markup text which can have entity markers."
+          }
+        ]
+      }
+    },
+    "QnAGenParameters": {
+      "type": "object",
+      "description": "Parameters for configuring QnA Generation steps and limiting the output.",
+      "additionalProperties": false,
+      "properties": {
+        "fetchRawContent": {
+          "type": "boolean",
+          "description": "Enable to fetch raw contents of document in the result."
+        },
+        "useAnswerMarkedInContent": {
+          "type": "boolean",
+          "description": "Specify this as true - if the documents have marker tags &lt;s&gt; and &lt;/s&gt; for highlighted answers. Example: Satya Nadella was born in &lt;s&gt; Hyderabad &lt;/s&gt; of present-day Telangana, India into a Telugu-speaking Hindu family."
+        },
+        "maxQnAs": {
+          "type": "integer",
+          "description": "Maximum number of QnAs to be generated.",
+          "format": "int32",
+          "maximum": 30,
+          "minimum": 1
+        }
+      }
+    },
+    "GeneratedQnAs": {
+      "type": "array",
+      "description": "An array of QnAs generated from a document in the job.",
+      "items": {
+        "$ref": "#/definitions/GeneratedQnA"
+      }
+    },
+    "GeneratedQnA": {
+      "type": "object",
+      "properties": {
+        "questions": {
+          "type": "string",
+          "description": "A question generated for a specific answer from a document in the job."
+        },
+        "answer": {
+          "type": "string",
+          "description": "Entity or phrase which is the answer to the generated question from a particular document in the job."
+        },
+        "score": {
+          "type": "number",
+          "description": "Confidence score of a question generated against the answer in the document's content."
+        },
+        "confidence": {
```

Comment:
The property name is confusing, especially given that other Language APIs call `score` `confidenceScore`. So then this `confidence` property isn't clear for users of other Language APIs. Given possible values of "low", "medium", and "high", what about something like `quality` or `confidenceBucket` or something more descriptive?